docs/_templates_system/templates.rst

```diff
@@ -38,9 +70,9 @@ gimmelist
 Tag :code:`gimmelist` have two required parameter and two optional:
 
  * (required) Name of variable available inside block: :code:`article`
- * (required) Keword :code:`from` and type of requested Meta's in collection: :code:`from articles` with filters passed to Meta Loader as extra parameters (:code:`start`, :code:`limit`, :code:`order`)
- * (optional) Keword :code:`with` and parameters for Meta Loader ex.: :code:`with {foo: 'bar', param1: 'value1'}`
- * (optional) Keword :code:`if` and expresion used for results filtering
+ * (required) Keyword :code:`from` and type of requested Meta's in collection: :code:`from articles` with filters passed to Meta Loader as extra parameters (:code:`start`, :code:`limit`, :code:`order`)
+ * (optional) Keyword :code:`with` and parameters for Meta Loader ex.: :code:`with {foo: 'bar', param1: 'value1'}`
+ * (optional) Keyword :code:`if` and expression used for results filtering
```

Comment:
but gimmelist don't render anything, it's just loop, and you can render something based on loop items inside.